### ISSUE TITLE
Automatic releases

### ISSUE THREAD

**Issue by altinnadmin** [Altinn/altinn2-convert#3]
We should add a GitHub Action that automatically builds this tool for windows/mac/linux and creates a new release with links when a PR is merged.

**Comment by ivarne** [comment]
Windows only because of CabLib.dll

**Comment by ivarne** [comment]
How hard would it be to take all the zip files currently deployed to production (altinn2) and put them in a storage account? Then we can have function app and just test everything?

**Comment by altinnadmin** [comment]
> Windows only because of CabLib.dll

Good point! If forgot about CabLib. Shouldn't be a biggie since we're porting from a Win-only tool.

> How hard would it be to take all the zip files currently deployed to production (altinn2) and put them in a storage account?

Shouldn't be very hard, all the zips are stored in an SQL Server table in TUL.

**Comment by ivarne** [comment]
Great I'll look into deploying this to a function app with a storage account trigger. Seems much more fun than to manually download all 120 KRT packages from TUL.
